components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/pom.xml

```diff
@@ -127,14 +127,15 @@
     <build>
 
         <plugins>
-            <!-- <plugin>
+            <plugin>
                 <groupId>org.wso2.maven.plugins</groupId>
                 <artifactId>swagger2msf4j-maven-plugin</artifactId>
                 <version>1.0-SNAPSHOT</version>
                 <configuration>
                     <inputSpec>${project.basedir}/src/main/resources/publisher-api.yaml</inputSpec>
+                    <autoEnableOptions>true</autoEnableOptions>
                 </configuration>
-            </plugin> -->
+            </plugin>
```

Comment:
Please keep commenting the swagger2msf4j-maven-plugin.

Reply:
Fixed with https://github.com/RenukaFernando/carbon-apimgt/commit/aa83fd7a66865b2dd15f6041bed648361f036494